sdk/migrationassessment/Azure.ResourceManager.Migration.Assessment/tests/MigrationAssessmentManagementTestEnvironment.cs

```diff
@@ -9,7 +9,5 @@ namespace Azure.ResourceManager.Migration.Assessment.Tests
 {
     public class MigrationAssessmentManagementTestEnvironment : TestEnvironment
     {
-        public override TokenCredential Credential =>
```

Comment:
This should not be needed and it is obsolete causing CI failure [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4699528&view=logs&j=84157c7d-47ab-5ac6-95b3-1e624ac86330&t=6c13f8e4-6d0f-54ca-28da-6a546516b3c5&l=113)